.eslintrc

```diff
@@ -0,0 +1,44 @@
+{
+  "root": true,
+  "extends": "airbnb",
+  "plugins": ["import", "react", "jsx-a11y" ],
+  "parser": "babel-eslint",
+  "env": {
+    "node": true,
+    "es6": true,
+  },
+  "ecmaFeatures": { "jsx": true },
+  "globals": {},
+  "rules": {
```

Comment:
Are you planning to set any `jsx-a11y` accessibility [suported rules](https://github.com/evcohen/eslint-plugin-jsx-a11y#supported-rules)?

Reply:
No, corrected.